libredex/Trace.cpp

```diff
@@ -129,7 +129,7 @@ struct Tracer {
     for (const char* tok = strtok(tracespec, sep); tok != nullptr;
          tok = strtok(nullptr, sep)) {
       auto level = strtol(tok, nullptr, 10);
-      if (level) {
+      if (level || *tok == '0') {
```

Comment:
Shouldn't this rather be `strcmp`? This is somewhat hackish. :-)

Reply:
True, this should be strcmp!

On another note, hackish code from @danzimm ? Who would've expected that! (cc @wsanville )